settings/R/create.multisite.settings.r

```diff
@@ -0,0 +1,55 @@
+##' @export
+createMultiSiteSettings <- function(template.settings, site.ids, out.dir) {
+  if(missing(out.dir)) {
+    if(is.null(template.settings$outdir)) {
+      warning("No out.dir set!")
+      out.dir <- NULL
+    } else {
+      out.dir <- template.settings$outdir
+    }
+  }
+  
+  template.settings <- .set.out.dir(template.settings, out.dir)
+  
+  run.settings <- lapply(site.ids, .get.run.settings, template.settings=template.settings)
+  
+  template.settings[["run", global=FALSE]] <- run.settings
+  return(template.settings)
+}
+
+##' @export
+.get.run.settings <- function(template.settings, site.id) {
+  start.date = template.settings$run$start.date
```

Comment:
This one really needs more documentation, as it's really unintuitive that this small subset of the XML counts as the run settings

Please add a section to the gitbook explaining that if the run settings are expanded in the future which functions need to be updated, otherwise I could see this function silently breaking lots of stuff when someone expands the run definition and then has their new parameters disappear.